src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -2154,6 +2154,15 @@ void ApplicationManagerImpl::updateRequestTimeout(
       connection_key, mobile_correlation_id, new_timeout_value);
 }
 
+bool ApplicationManagerImpl::IsUpdateRequestTimeoutRequired(
```

Comment:
@LitvinenkoIra signature have cont refs parameters, but in implementation pure ints

Reply:
@LuxoftAKutsan fixed 278050fc416a577487a1bd2acbf1eedfdbea3c37